<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamWindow</class>
 <widget class="QWidget" name="ParamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="setFillHoles">
         <property name="text">
          <string>Wype≈Çnij dziury</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="setFilter">
         <property name="text">
          <string>filtr Gaussa</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Progowanie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="TreshildSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showTreshVal">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Kernel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="KernelSlider">
         <property name="maximum">
          <number>15</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showKernValue">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
